config/config.exs

```diff
@@ -33,7 +33,7 @@ config :joken, default_signer: "secret"
 # Configures the mechanism for erlang node clustering
 config :libcluster,
   topologies: [
-    local: [strategy: Cluster.Strategy.Gossip]
+    # local: [strategy: Cluster.Strategy.Gossip]
```

Comment:
@taylordowns2000 why is this commented out?

Reply:
@elias-ba can you change this back for this PR.

Reply:
Done